### PR TITLE
Discord Credentials update

### PR DESCRIPTION
**Pull request by gandreini**

### CODE REVIEW

File: docs/integrations/builtin/credentials/discord.md [hunk 1]
```diff
@@ -5,22 +5,74 @@ contentType: integration
...
+
+- **Bot Token:** Bots are added to a Discord server and can interact with users on all the server’s channels. They can **manage channels**, **send and retrieve messages**, retrieve the **list of all users**, and **change their roles**. If you need to build an interactive, complex, or multi-step workflow a Bot is the way to go.
+- **OAuth2:** OAuth credentials offer the same functionalities of the Bot Token. The only difference is that is simplifies the installation of the Bot on your server.
+- **Webhook:** Webhooks are added to a single channel in a server, they are a simple way to **post messages to a channel**. They do not require a Bot user or authentication, making them easier to set up for simple use cases. However, they can't listen or respond to user requests or commands. If you need a straightforward way to send messages to a channel, without the need for interaction or feedback, a Webhook is sufficient.
+
+## Bot Token manual installation
+
+Use this method if you want to add the Bot to your Discord server manually (without using OAuth2)
+
+1. Go to the [Applications](https://discord.com/developers/applications) page on the Discord developers portal
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/discord.md:27]:
```suggestion
1. Go to the [Applications](https://discord.com/developers/applications){:target=_blank .external-link} page on the Discord developers portal
```

File: docs/integrations/builtin/credentials/discord.md [hunk 2]
```diff
@@ -5,22 +5,74 @@ contentType: integration
...
+9. Copy the URL clicking on **Copy** In **Generated URL** at the bottom
+10. Open the copied URL in a new browser tab and follow the instructions to add the Bot to a server
+    - In **Add to Server** select the server you want to add the Bot to, and click **Continue**
+    - In the following page, click on **Authorize**
+
+## Bot Token OAuth2 installation
+
+This process simplifies the addition of your Bot to the Discord server.
+
+1. Go to the [Applications](https://discord.com/developers/applications) page on the Discord developers portal
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/discord.md:58]:
```suggestion
1. Go to the [Applications](https://discord.com/developers/applications){:target=_blank .external-link} page on the Discord developers portal
```

File: docs/integrations/builtin/credentials/discord.md [hunk 3]
```diff
@@ -5,22 +5,74 @@ contentType: integration
...
 
 ## Prerequisites
 
-Create a [Discord](https://www.discord.com/) account.
+Create a [Discord](https://www.discord.com/) account
 
-## Creating a webhook in Discord
+## Which type of credentials to use
+
+- **Bot Token:** Bots are added to a Discord server and can interact with users on all the server’s channels. They can **manage channels**, **send and retrieve messages**, retrieve the **list of all users**, and **change their roles**. If you need to build an interactive, complex, or multi-step workflow a Bot is the way to go.
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/discord.md:19]:
```suggestion
- **Bot Token:** bots are added to a Discord server and can interact with users on all the server’s channels. They can manage channels, send and retrieve messages, retrieve the list of all users, and change their roles. If you need to build an interactive, complex, or multi-step workflow use a bot.
```

File: docs/integrations/builtin/credentials/discord.md [hunk 4]
```diff
@@ -5,22 +5,74 @@ contentType: integration
...
+- **OAuth2:** OAuth credentials offer the same functionalities of the Bot Token. The only difference is that is simplifies the installation of the Bot on your server.
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/discord.md:20]:
```suggestion
- **OAuth2:** OAuth credentials offer the same functionalities as the Bot Token. The difference is that it simplifies the installation of the bot on your server.
```

File: docs/integrations/builtin/credentials/discord.md [hunk 5]
```diff
@@ -5,22 +5,74 @@ contentType: integration
...
+- **Webhook:** Webhooks are added to a single channel in a server, they are a simple way to **post messages to a channel**. They do not require a Bot user or authentication, making them easier to set up for simple use cases. However, they can't listen or respond to user requests or commands. If you need a straightforward way to send messages to a channel, without the need for interaction or feedback, a Webhook is sufficient.
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/discord.md:21]:
```suggestion
- **Webhook:** webhooks are added to a single channel in a server. They're a simple way to post messages to a channel. They do not require a bot user or authentication, making them easier to set up for simple use cases. However, they can't listen or respond to user requests or commands. If you need a straightforward way to send messages to a channel, without the need for interaction or feedback, a webhook is sufficient.
```

File: docs/integrations/builtin/credentials/discord.md [hunk 6]
```diff
@@ -5,22 +5,74 @@ contentType: integration
...
+
+## Bot Token manual installation
+
+Use this method if you want to add the Bot to your Discord server manually (without using OAuth2)
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/discord.md:25]:
```suggestion
Use this method if you want to add the bot to your Discord server manually (without using OAuth2).
```

File: docs/integrations/builtin/credentials/discord.md [hunk 7]
```diff
@@ -5,22 +5,74 @@ contentType: integration
...
+
+1. Go to the [Applications](https://discord.com/developers/applications) page on the Discord developers portal
+2. Select or create an application
+3. In the application page, go to **Bot**, and in **Privileged Gateway Intents**, activate **Required for your bot to receive events listed under GUILD_MEMBERS**
+4. Generate a Bot token by clicking on **Reset Token**
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/discord.md:30]:
```suggestion
4. Generate a bot token by clicking on **Reset Token**
```

File: docs/integrations/builtin/credentials/discord.md [hunk 8]
```diff
@@ -5,22 +5,74 @@ contentType: integration
...
+
+This process simplifies the addition of your Bot to the Discord server.
+
+1. Go to the [Applications](https://discord.com/developers/applications) page on the Discord developers portal
+2. Select or create an application
+3. ? In the application page, go to **Bot**, and in **Privileged Gateway Intents**, activate **Required for your bot to receive events listed under GUILD_MEMBERS**
+4. Copy the CLIENT ID in the **OAuth2** page and paste it into the Client ID parameter in the n8n's Discord OAuth2 API credential modal
+5. Generate a CLIENT SECRET in the **OAuth2** page and paste it into the Client Secret parameter in the n8n's Discord OAuth2 API credential modal
+6. Copy the OAuth Redirect URL in n8n and add it to the **Redirects** list in your Discord app and save
+7. ? In the Discord developer portal, go to the Bot page  of your app and copy the Bot Token (Generate one if you don’t have it yet)
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/discord.md:64]:
```suggestion
7. In the Discord developer portal, go to the Bot page  of your app and copy the Bot Token (Generate one if you don’t have it yet)
```

File: docs/integrations/builtin/credentials/discord.md [hunk 9]
```diff
@@ -5,22 +5,74 @@ contentType: integration
...
+8. ? Paste the Bot Token in n8n
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/discord.md:65]:
```suggestion
8. Paste the Bot Token in n8n
```

File: docs/integrations/builtin/credentials/discord.md [hunk 10]
```diff
@@ -5,22 +5,74 @@ contentType: integration
...
+9. Click **Connect my account** in the n8n' Discord OAuth2 API credential modal
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/discord.md:66]:
```suggestion
9. Select **Connect my account** in the n8n Discord OAuth2 API credential modal
```

File: docs/integrations/builtin/credentials/discord.md [hunk 11]
```diff
@@ -5,22 +5,74 @@ contentType: integration
...
+10. A window will show up where you’ll be required to select the Discord server and the permission you want to set for your Bot
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/discord.md:67]:
```suggestion
10. n8n displays a window where must select the Discord server and the permission you want to set for your bot
```

File: docs/integrations/builtin/credentials/discord.md [hunk 12]
```diff
@@ -5,22 +5,74 @@ contentType: integration
...
+
+## Creating a Webhook
 
 1. To create a webhook in Discord that sends content to your channel, go to the settings of your channel.
 2. Select 'Integrations' from the sidebar.
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/discord.md:72]:
```suggestion
2. Select **Integrations** from the sidebar.
```

File: docs/integrations/builtin/credentials/discord.md [hunk 13]
```diff
@@ -5,22 +5,74 @@ contentType: integration
...
 3. Click on the ***Create Webhook*** button.
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/discord.md:73]:
```suggestion
3. Select **Create Webhook**.
```

File: docs/integrations/builtin/credentials/discord.md [hunk 14]
```diff
@@ -5,22 +5,74 @@ contentType: integration
...
 4. Name your bot, and click on the ***Copy Webhook URL*** button to copy the webhook URL.
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/discord.md:74]:
```suggestion
4. Name your bot, and select **Copy Webhook URL** to copy the webhook URL.
```

File: docs/integrations/builtin/credentials/discord.md [hunk 15]
```diff
@@ -5,22 +5,74 @@ contentType: integration
...
+3. ? In the application page, go to **Bot**, and in **Privileged Gateway Intents**, activate **Required for your bot to receive events listed under GUILD_MEMBERS**
+4. Copy the CLIENT ID in the **OAuth2** page and paste it into the Client ID parameter in the n8n's Discord OAuth2 API credential modal
+5. Generate a CLIENT SECRET in the **OAuth2** page and paste it into the Client Secret parameter in the n8n's Discord OAuth2 API credential modal
+6. Copy the OAuth Redirect URL in n8n and add it to the **Redirects** list in your Discord app and save
+7. ? In the Discord developer portal, go to the Bot page  of your app and copy the Bot Token (Generate one if you don’t have it yet)
+8. ? Paste the Bot Token in n8n
+9. Click **Connect my account** in the n8n' Discord OAuth2 API credential modal
+10. A window will show up where you’ll be required to select the Discord server and the permission you want to set for your Bot
+
+## Creating a Webhook
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/discord.md:69]:
```suggestion
## Creating a webhook
```

**Comment by gandreini** [comment]
@StarfallProjects here's the updated documentation for the Discord creds, including Both and OAuth

**Review by StarfallProjects — APPROVED** [review]

**Pull request merged**